docs/src/main/sphinx/release/release-385.md

```diff
@@ -0,0 +1,37 @@
+# Release 385 (dd June 2022)
+
+## General
+
+* Introduce `json_array` and `json_object` [JSON functions](/functions/json). ({issue}`9081`)
+* Support all types that can be cast to `varchar` as parameters for the JSON
+  path. ({issue}`12682`)
+* Allow `CREATE TABLE LIKE` a table from a different catalog when explicitly
+  excluding properties. ({issue}`3171`)
+* Reduce `Exceeded limit of N open writers for partitions` errors when
+  fault-tolerant execution is enabled. ({issue}`12721`)
+
+## Delta Lake connector
+
+* Add support for the `appendOnly` field. ({issue}`12635`)
+* Add support for column comments when creating a table or a column. ({issue}`12455`, {issue}`12715`))
+
+## Hive connector
+
+* Allow cancelling a query on a transactional table if it is waiting for a lock. ({issue}`11798`)
+* Propagate compression codec selected by `hive.compression-codec` config
+  property or `compression_codec` session property to Avro file writer. ({issue}`12639`)
+
+## Iceberg connector
+
+* Improve query performance when a table consists of many small files. ({issue}`12579`)
+* Improve query performance when performing a delete or update. ({issue}`12671`)
```

Comment:
@findepi is this reasonable?

Reply:
> Improve query performance when performing a delete or update

not really. the delete/update performance is the same as it used to be
however, delete/update now produces files that are more efficient to query later